brainstormforce/ultimate-addons-for-gutenberg
# Fix: Styling not applying on frontend when block used in widget area.

### Description

- https://app.startinfinity.com/b/Mi5zu3VRVzk/3sAgbH1y7qY/f92da082-c136-4358-8d2f-2a5947267ea9?redirect=true&view=6e170ea7-8708-4546-8b82-23d2eebdf694&t=attributes

### Screenshots

- Output - https://share.getcloudapp.com/6quEexY9

### Types of changes




- Bug fix (non-breaking change which fixes an issue

### How has this been tested?

- Check is there any notice/error visible in the widget area when UAG is activated.
- Drag and drop block add styling and enable sidebar or footer.
- Check blocks on front is it visible, is styling applying to block properly.

### Checklist:
